rosdep/base.yaml

```diff
@@ -2389,6 +2389,11 @@ libpqxx:
     trusty: [libpqxx-4.0]
     utopic: [libpqxx-4.0]
     vivid: [libpqxx-4.0]
+    wily: [libpqxx-4.0]
+    xenial: [libpqxx-4.0]
+    yakkety: [libpqxx-4.0v5]
+    zesty: [libpqxx-4.0v5]
+    artful: [libpqxx-4.0v5]
```

Comment:
In order to use a custom order the code would need to implement the serialization manually as well as know a list of all platforms (which would need to be continuously updated).

For now the ordering is alphabetical. Please update the patch accordingly in order to be mergable.